contracts/cw-core/src/contract.rs

```diff
@@ -717,10 +789,31 @@ pub fn reply(deps: DepsMut, _env: Env, msg: Reply) -> Result<Response, ContractE
         PROPOSAL_MODULE_REPLY_ID => {
             let res = parse_reply_instantiate_data(msg)?;
             let prop_module_addr = deps.api.addr_validate(&res.contract_address)?;
-            PROPOSAL_MODULES.save(deps.storage, prop_module_addr, &Empty {})?;
+            let total_module_count = TOTAL_PROPOSAL_MODULE_COUNT
+                .may_load(deps.storage)?
```

Comment:
Instead of `may_load` here, how would you feel about setting this to zero during instantiation?

Otherwise, there is a period of time where this has no value when modules are being instantiated. If they were to query back to the DAO for this info it would error as there is no value.

Reply:
good point!